flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -4376,7 +4373,7 @@ protected void testConcurrentMapIfQueryable() throws Exception {
 
                 StateTable<?, ?, ?> stateTable =
                         ((AbstractHeapState<?, ?, ?>) kvState).getStateTable();
-                checkConcurrentStateTable(stateTable, numberOfKeyGroups);
+                assertNotNull("State not set", stateTable);
```

Comment:
I think `assertNotNull("State not set", ((AbstractHeapState<?, ?, ?>) kvState).getStateTable());` would be better.

Reply:
Sure, I've changed all three occurrences.